inc/namespace.php

```diff
@@ -182,7 +182,7 @@ function add_login_logo() {
 	?>
 	<style>
 		.login h1 a {
-			background-image: url('<?php echo esc_url( site_url( $logo ) ); ?>');
+			background-image: url('<?php echo esc_url( get_site_url( 1, $logo ) ); ?>');
```

Comment:
Can this use `get_main_site_id( get_main_network_id() )` instead? I think 1 is generally correct but in some really bizarre circumstances it's not always the case

Reply:
```suggestion
			background-image: url('<?php echo esc_url( get_site_url( get_main_site_id( get_main_network_id() ), $logo ) ); ?>');
```